src/cli/services/linter/linter.ts

```diff
@@ -91,5 +91,20 @@ export async function lint(documents: Array<number | string>, flags: ILintConfig
     );
   }
 
+  filesNotFound.forEach(fileNotFound => {
+    const errorFileNotFound: IRuleResult = {
+      code: 'file-not-found',
+      message: 'File not found: ' + fileNotFound,
+      path: [],
+      severity: 0,
+      source: 'File not found',
+      range: {
+        start: { line: 1, character: 1 },
```

Comment:
Lines and characters are zero-based.
```suggestion
        start: { line: 0, character: 0 },
```